app/assets/stylesheets/component_library_overrides.css

```diff
@@ -21,3 +21,7 @@
 .navbar-nav {
   --bs-nav-link-color: black;
 }
+
+.dropdown-menu li {
+  display: block!important;
```

Comment:
Would you mind leaving a comment here about why we need the `!important` directive? also, please add a space in between.

Reply:
This has been hanging out so long Nick has fixed it in the component library, so I removed the code. It won't be an issue once a new release is cut and for now I think it's fine if the dropdown appears horizontally, especially since I think we will get a new release before this goes into production.